inventory/byo/hosts.ose.example

```diff
@@ -98,9 +98,9 @@ openshift_release=v3.7
 # Add additional, insecure, and blocked registries to global docker configuration
 # For enterprise deployment types we ensure that registry.access.redhat.com is
 # included if you do not include it
-#openshift_docker_additional_registries=registry.example.com
-#openshift_docker_insecure_registries=registry.example.com
-#openshift_docker_blocked_registries=registry.hacker.com
+#openshift_docker_additional_registries=['registry.example.com']
+#openshift_docker_insecure_registries=['registry.example.com']
+#openshift_docker_blocked_registries=['registry.hacker.com']
```

Comment:
Is this conversion to a list mandatory? We can't force people to change the format unless there's no way around it.

Reply:
We discussed this in IRC a bit.

It seems these examples were in error.  Everything, including our unit tests, expects this to be a list.  This should probably be a list, it doesn't appear to support CSV strings at this time either.

Reply:
In my test inventory I've always had the following, I'll run through a quick test with this PR and my standard inventory.

`openshift_docker_additional_registries="registry1.redhat.com,registry2.redhat.com:8888"`

Reply:
```
TASK [docker : debug] *************************************************************************************************************************************************************************************************************************
Tuesday 19 September 2017  17:40:15 -0400 (0:00:00.077)       0:00:25.384 ***** 
ok: [ose3-master.example.com] => {
    "msg": "insecure_registries registry.ops.openshift.com,brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888"
}

TASK [docker : Set registry params] ***********************************************************************************************************************************************************************************************************
Tuesday 19 September 2017  17:40:15 -0400 (0:00:00.077)       0:00:25.461 ***** 
fatal: [ose3-master.example.com]: FAILED! => {"failed": true, "msg": "|failed expects first param is a list"}
```

This code was responsible for splitting it when it was provided as a comma separated list https://github.com/openshift/openshift-ansible/pull/5165/files#diff-8c83287a125220bc46f58c79f26c6670L2275

Reply:
@sdodson I added some new variables in roles/docker/defaults/main.yml to safely convert any csv values (include just a single item) to a list.  If the values is already a list, it will remain as-is.